test/unit/batching/vendor-lines/get-payment-type.test.js

```diff
@@ -0,0 +1,108 @@
+const { BPS, CS, FDMR, LumpSums, SFI, SFIP, VetVisits, IMPS, ES } = require('../../../../app/constants/schemes')
+const { getPaymentType } = require('../../../../app/batching/vendor-lines/get-payment-type')
+
+describe('get payment type tests', () => {
+  let paymentType
```

Comment:
Does the let need to be defined in the describe for some reason?

Reply:
No reason and no idea why I put it there.  Moved.